src/state.ts

```diff
@@ -15,17 +22,18 @@ export async function refreshPiggybankState(rpc: ConcordiumGRPCClient, contract:
     console.debug(`Refreshing piggybank state for contract ${contract.index.toString()}`);
     const { version, name, index, methods } = contract;
 
-    const expectedMethods = ['insert', 'smash', 'view'].map((m) => `${name}.${m}`);
-    if (!expectedMethods.every(methods.includes.bind(methods))) {
+    const expectedMethods = ['insert', 'smash', 'view'].map((m) => `${ContractName.toString(name)}.${m}`);
```

Comment:
Yes, in general I regret that we didn't implement `toString` to make it easier to use in string interpolation, and `toJSON` to unwrap the inner values to an expected format. But adding this will result in breaking changes in the browser wallet due to `instanceof` checks (which in hindsight was also a bad decision on our part).

It's definitely something we should add when hopefully we'll update how transaction payloads are serialized with the implementation of https://github.com/Concordium/concordium-node-sdk-js/issues/307